changes

teodorakocan opened the pull request:
- added new function "reloadPage" because the elements are listed reversely and clicking on delete button wrong element is deleted
- changed the name of some constants
- removed grid from css and used only flex

teodorakocan committed:
changes

- added new function "reloadPage" because the elements are listed reversely and clicking on delete button wrong element is deleted
-changed the name of some constants
-removed grid from css and used only flex

AirmiX commented on index.html:
```diff
@@ -1,33 +1,34 @@
+            <script src='index.js'></script>
 </body>
+
 </html>
```
Let's go with `'` in JS and `"` in HTML.

AirmiX commented on index.js:
```diff
@@ -7,12 +7,13 @@ let numberOfActiveTasks = 0;
 const buttonAll = document.getElementById('all');
 const buttonCompleted = document.getElementById('completed');
 const buttonActive = document.getElementById(ACTIVE_CLASS);
```
Change to 'active'. It is not the same thing as `ACTIVE_CLASS`

AirmiX commented on index.js:
```diff
@@ -25,20 +26,16 @@ input.addEventListener('keyup', function (event) {
+    const tasksString = localStorage.getItem('tasks') || '[]';
+    tasks = JSON.parse(tasksString);
+    tasks
+        .forEach(function(currentTask) {
```
Use arrow function for the callback functions (since you already used it in few places, so to be consistent)

AirmiX replied:
Line 16 as well

AirmiX commented on style.css:
```diff
@@ -1,135 +1,129 @@
 
-.taskarea:hover{
-  overflow: visible;
+.trimName {
```
Good idea to have it as a separate class 👏. Rename to `trimText`, since it works for text in general and can be reused

AirmiX commented on index.js:
```diff
@@ -83,7 +76,10 @@ function crossTask(value) {
+    reloadPage();
+}
 
+function reloadPage() {
```
Rename to `reloadTasks`, with this name I thought the page is actually reloading like before

teodorakocan committed:
changes

- changed ' to " in html file
- changed ACTIVE_CLASS in document.getElemenetById(ACTIVE_CLASS) to document.getElemenetById('active')
- in all callback functions put arrow
- renamed reloadPage function to relaodTasks()

AirmiX commented on index.js:
```diff
@@ -52,23 +49,19 @@ function addTask(name) {
     localStorage.setItem(TASK_KEY, JSON.stringify(tasks));
-    renderTasks();
     renderNumberOfActiveTasks();
+    reloadPage();
```
Does this function exist?

AirmiX commented on style.css:
```diff
@@ -1,135 +1,129 @@
 
-.taskarea:hover{
-  overflow: visible;
+.trimText {
```
This will override the previous class

teodorakocan committed:
corrected errors
